### ISSUE TITLE
Vanilla torch recipe not successfully removed

### ISSUE THREAD

**Issue by sableseyler** [MattCzyr/RealisticTorches#31]
On 1.15.2, the standard stick+coal torch recipe produced vanilla torches. Both the realistic torch recipe and the vanilla torch recipe are present. No error messages from this mod in the log.

**Comment by firefancy** [comment]
I'm having this problem too. Did you start a new save, or did you resume a save that you had before you installed the mod? In the issue that I posted, I noticed that when I resumed playing a save that I had before installing the mod, this mod would do exactly this. When I started a brand new save, the mod worked as intended.

**Comment by Rotzo** [comment]
i have the same problem, even on a new save.

**Comment by Lystraeus** [comment]
Bug still exists on a resumed save in version 3.1.0 for 1.16.4

**Comment by MattCzyr** [comment]
Might be fixed in this beta version -- haven't been able to recreate it myself. Try it out and let me know:

https://www.dropbox.com/s/0r3085jrdb3sf2z/RealisticTorches-1.16.4-3.1.2-beta.jar?dl=0

**Comment by EleventhOne** [comment]
Dear developer, the **bug persists on 1.6.5 version: 3.1.2** just letting you know, take your time

**Comment by Winddbourne** [comment]
Yup, I can't even pull an unlit torch out of the creative inventory.  It doesn't seem to exist so I can't even fix the problem with a data pack while you work on things. Hope you get this taken care of ASAP.

**Comment by MattCzyr** [comment]
> Yup, I can't even pull an unlit torch out of the creative inventory. It doesn't seem to exist so I can't even fix the problem with a data pack while you work on things. Hope you get this taken care of ASAP.

Unlit torches do exist, just not in the creative inventory. The ID is `realistictorches:unlit_torch`.

**Comment by NovaLynxie** [comment]
Just saw this issue. I have also got JEI installed and it does not show it at all there. Only the lit torch and vanilla torch shows up.
Doing a search for realistictorches in JEI shows everything except unlit torches. It is as if it does not exist at all.

**Comment by qwitwa** [comment]
A workaround for now - remove the vanilla recipe with CraftTweaker. Here's a one line CraftTweaker script that still allows creating torches with glowstone paste:

`craftingTable.removeByRegex("minecraft:torch");
`

**Comment by lunafoxfire** [comment]
Can confirm that this problem still exists. It can be worked around with datapacks though. Just add an empty recipe at `data/minecraft/recipes/torch.json`

**Comment by NixityNull** [comment]
> A workaround for now - remove the vanilla recipe with CraftTweaker. Here's a one line CraftTweaker script that still allows creating torches with glowstone paste:
> 
> `craftingTable.removeByRegex("minecraft:torch"); `

What if you don't know how to use crafttweaker? I find it super confusing and frustrating because it doesn't have a config file and idk how to get to the file to edit.

**Comment by marioessig** [comment]
> A workaround for now - remove the vanilla recipe with CraftTweaker. Here's a one line CraftTweaker script that still allows creating torches with glowstone paste:
> 
> `craftingTable.removeByRegex("minecraft:torch");`

Does not work while on server. It does work when playing in a singleplayer world.
